google/cloud/firestore_v1/base_query.py

```diff
@@ -972,6 +974,15 @@ def _to_protobuf(self) -> StructuredQuery:
             query_kwargs["limit"] = wrappers_pb2.Int32Value(value=self._limit)
         return query.StructuredQuery(**query_kwargs)
 
+    def find_nearest(
+        self,
+        vector_field: str,
+        queryVector: Vector,
```

Comment:
it might be good to accept a list/np array here as well, and convert them to a Vector if needed. But that can be a future consideration

Reply:
Nodejs-firestorealso supports an array of numbers for the queryVector